jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/common/ColorSpinner.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.jdi.light.ui.bootstrap.elements.common;
+
+import com.epam.jdi.light.asserts.generic.UIAssert;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ColorSpinner extends UIBaseElement<UIAssert> {
+
+    public String getColor(){
+        String str = core().getAttribute("class");
+        Pattern pattern = Pattern.compile("text-.+");
+        Matcher matcher = pattern.matcher(str);
+        String result;
+        if (matcher.find()) {
+            result = str.substring(matcher.start(), matcher.end());
+        }
+        else{
+            result = "The color is not set";
```

Comment:
it's bad idea to return some text when there is no color. you can return the same value as getText() method when there is no text (in my opinion, it's empty string, but I'm not sure)

Reply:
You are right. I found out that the method getText(), in the end, calls the construction SMART_GET_TEXT (https://github.com/jdi-testing/jdi-light/blob/bootstrap/jdi-light/src/main/java/com/epam/jdi/light/elements/common/UIElement.java), which returns an empty string ""  in the worst case.